javascript-loops-practice/get-student-names.js

```diff
@@ -0,0 +1,10 @@
+/* exported getStudentNames */
+function getStudentNames(students) {
+  var values = [];
+  students.forEach(function (students) {
+    for (var x in students) {
+      values.push(students[x]);
+    }
```

Comment:
You don't need all properties in the object, you just need the `name` property's value.